features/collections/icon.feature

```diff
@@ -17,8 +17,10 @@ Feature: User icons
     And I press "Update"
   Then I should see the image "alt" text "Some test description"
 
-  Scenario: Add an icon to a collection
+  Scenario: Add and remove an icon to a collection
```

Comment:
It is worth noting that the test is useful as it is as it fails on an exception if we don't have :delete_icon